src/commcare_cloud/commands/terraform/templates/commcarehq.tf.j2

```diff
@@ -248,6 +248,12 @@ module "logshipping" {
   account_id = {{ account_id|tojson }}
 }
 
+module "autoscaling" {
+  source = "./modules/autoscaling"
+  release_bucket = var.release_bucket
+  commcare_server_role_id = module.server_iam_role.commcare_server_role_id
+}
```

Comment:
If we merge this as is, I think this will error on production, because this variable is `None`. I think probably you want to "hide" this addition from environments other than staging until you're ready to apply it, by wrapping it in

```
{% if release_bucket %}
...
{% endif %}
```